### DISCUSSION TITLE
PT-BR update

### DISCUSSION THREAD

**Discussion by joaomino** [aclap-dev/video-downloadhelper#904] [Language & Translation]
Non translated strings;
 _v9_chrome_premium_hls_ string fix. Missing the word _minutes_.
[v9_chrome_premium_hls-pt_BR.txt](https://github.com/user-attachments/files/15902761/v9_chrome_premium_hls-pt_BR.txt)
[non-translated-pt_BR.txt](https://github.com/user-attachments/files/15902763/non-translated-pt_BR.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
